ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2566,6 +2576,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi
+- as: vsphere-8-upi-p3-f28
```

Comment:
I guess we can reuse the above `vsphere-upi-p3-f28` job, just switch `cluster_profile` to `vsphere-8`, and rename it to `vsphere-8-upi-p2-f7`

Reply:
if that case, now i just remove the vsphere-upi-p3-f28 job and vsphere-ipi-ovn-etcd-encryption-p2-f7 job, is that ok?

Reply:
Hmm, I guess it not "remove", but "modify" the exiting ones.

Reply:
yes, because i have added extra two jobs(vsphere-8-upi-p2-f7,vsphere-8-ipi-p2-f7) in the file. you mean to modify the previous two jobs(vsphere-upi-p3-f28 , vsphere-ipi-ovn-etcd-encryption-p2-f7). so my understanding is now i just need to remove these two old jobs, because i have already add two new jobs.

Reply:
> you mean to modify the previous two jobs(vsphere-upi-p3-f28 , vsphere-ipi-ovn-etcd-encryption-p2-f7).
`vsphere-upi-p3-f28` can be removed, but need to modify `vsphere-ipi-ovn-etcd-encryption-p2-f7` to switch over cluster profile to vsphere-8, because the workflow also covered `ovn-etcd-encryption` test points, not only vsphere-ipi.

Reply:
ok got it.

Reply:
I think we can increase the test frequency to `f7`, so the job name should be `vsphere-8-upi-p2-f7`